src/Simplex/Chat/Messages.hs

```diff
@@ -529,6 +529,7 @@ data CIFileStatus (d :: MsgDirection) where
   CIFSRcvInvitation :: CIFileStatus 'MDRcv
   CIFSRcvAccepted :: CIFileStatus 'MDRcv
   CIFSRcvTransfer :: {rcvProgress :: Int64, rcvTotal :: Int64} -> CIFileStatus 'MDRcv
+  CIFSRcvAborted :: CIFileStatus 'MDRcv
```

Comment:
```suggestion
  CIFSRcvNotApproved :: CIFileStatus 'MDRcv
```

Reply:
Aborted is unclear here, better to use the same term consistently

Reply:
this does happen when it's aborted though, could be reused for other errors possibly